simplerunner.lua

```diff
@@ -309,7 +309,7 @@ local opt_alias = {
 function opt_map.help()
     io.stdout:write[[
 
-Usage: simplerunner [OPTION]...  benchmark, count
+Usage: simplerunner [OPTIONS] [<benchmark_1> ... <benchmark_n>] [count]

```

Comment:
The trailing argument parsing was very simplistic it only accepts one benchmark name

Reply:
Ah yes, you're right. I'll fix the docs.